ci/jobs/pull_request.yml

```
@@ -0,0 +1,50 @@
+jobs:
+  - name: dataworks-analytical-service-infra-pr
+    plan:
+      - get: dataworks-analytical-service-infra-pr
+        trigger: true
+        version: every
+      - put: dataworks-analytical-service-infra-pr
+        params:
+          context: $BUILD_JOB_NAME
+          path: dataworks-analytical-service-infra-pr
+          status: pending
+      - get: dataworks-analytical-service-infra-client-release
+        version: { tag: '0.0.1' }
+      - .: (( inject meta.plan.terraform-bootstrap ))
+        params:
+          DEPLOY_PATH: infra
+        input_mapping:
+          dataworks-analytical-service-infra: dataworks-analytical-service-infra-pr
+      - task: terraform-plan-infra
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: ((terraform.repository))
+              version: ((terraform12.version))
+              tag: ((terraform12.version))
```

is now `((dataworks.terraform_version))`, as 12 is preferred.  For context TF11 is now, `((dataworks.terraform_11_version))`

```suggestion
              version: ((dataworks.terraform_version))
              tag: ((dataworks.terraform_version))
```